hack/pman/rules.yaml

```diff
@@ -0,0 +1,43 @@
+# note: if a folder ends with "*" it means it is recursive,
+# else it includes only the current directory files
+all:
+  - cluster-provision/gocli/*
+  - hack/kustomization/*
+  - cluster-provision/k8s
+  - cluster-provision
+  - cluster-up
+  - cluster-up/cluster
+  - cluster-up/hack
+  - hack
+  - publish.sh
+  - hack/pman/*
+none:
+  - cluster-provision/images/*
+  - cluster-provision/tools/*
+  - cluster-up/cluster/external
+  - cluster-up/cluster/local
+  - LICENSE
+  - Makefile
+  - OWNERS
+  - .gitignore
+# creates a specific rule for each folder, based on the foler base name (last element)
+# regex is always recursive
+# only last dir can be a regex
+# example: cluster-provision/k8s/[0-9].[0-9][0-9]*
+regex:
+  - cluster-provision/k8s/[0-9].[0-9][0-9]*
+  - cluster-up/cluster/k8s-[0-9].[0-9][0-9]*
+# regex_none will expand the list and assign to each rule the target "none"
+# key should be a directory ending with *
+# i.e cluster-up/cluster/k3d*
+regex_none:
+  - cluster-up/cluster/kind*
+  - cluster-up/cluster/k3d*
+# exclude supports only one value per pattern
+exclude:
+  - "pattern": "cluster-provision/centos8/*"
+    "exclude": "1.26-centos9"
```

Comment:
Would need to support multi value here (separated by ',' or a list).
Because a new 1.27 centos9 is going to be out soon
Same for "target" of "specific"